src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -52,11 +60,20 @@ const defaultProps = {
     onSelectRow: () => {},
     isFocused: false,
     style: null,
+    comment: '',
 };
 
 function OptionRowLHN(props) {
     const optionItem = SidebarUtils.getOptionData(props.reportID);
 
+    useEffect(() => {
+        if (!optionItem) return;
+        if (!optionItem.hasDraftComment && props.comment && props.comment.length > 0 && !props.isFocused) {
+            Report.setReportWithDraft(props.reportID, true);
+        }
+        // eslint-disable-next-line react-hooks/exhaustive-deps -- we don't want this effect to run again
```

Comment:
We can remove the `we don't want this effect to run again` since it's noticeable for the dev.

Reply:
@mollfpr okay, I just copied this from `src/Expensify.js`